app/services/school_data_exporter.rb

```diff
@@ -6,30 +6,41 @@ class SchoolDataExporter
 
   attr_reader :filename
 
-  def initialize(filename)
+  def initialize(filename = nil)
     @filename = filename
   end
 
   def export_schools(query = schools)
-    CSV.open(filename, 'w') do |csv|
-      csv << headings
-      query.find_each do |school|
-        csv << [
-          school.responsible_body.computacenter_identifier,
-          *build_name_fields_for(school),
-          school.address_1,
-          school.address_2,
-          school.address_3,
-          school.town,
-          school.postcode,
-        ]
+    if filename
```

Comment:
Do we still need the filename generation after this change?

Reply:
I guess it depends on how quickly CC transition to use list this and whether we need a fall back?

Reply:
yep fair enough, it's worth scheduling a separate card for cleaning this up shortly after CC transition to the new process.